src/utils/money.cljs

```diff
@@ -205,10 +205,14 @@
       (with-precision 2)
       str))
 
-(defn add
+(defn- add*
   [bn1 n2]
   (.add ^js bn1 n2))
 
```

Comment:
not for now, but these methods should probably all be covered in tests. We can add this as part of the post 1.27 workset 👍

Reply:
I added it (and explained why) in a previous PR.

https://github.com/status-im/status-mobile/pull/17920#discussion_r1394942416

I can add some tests in a following PR @J-Son89

Reply:
@ulisesmac - I was making no comment about the added function, I understand the use case. My point is more that we are using a bunch of (afaik) untested utilities that handle numbers. Tests would be great but we can wait until post 1.27 as let's focus on the delivering those features to the best quality as possible :)

Reply:
Oh yes, that's true!, we are relying on the functions in this namespace. They need to be tested.